client/src/reducers/feedbackReducer.ts

```diff
@@ -0,0 +1,35 @@
+import { Feedback } from 'types';
+import { uid } from 'utils/uid';
+
+export type ActionData = {
+  type: string;
+  payload?: {
+    name: string;
+    data?: string | number | readonly string[] | undefined;
+  };
+};
+export const CREATE_FEEDBACK_SUCCESS = 'CREATE_FEEDBACK_SUCCESS';
+export const FEEDBACK_FORM_RESET = 'FEEDBACK_FORM_RESET';
+
+const feedbackReducer = (state: Feedback, { type, payload }: ActionData) => {
+  switch (type) {
+    case CREATE_FEEDBACK_SUCCESS:
+      return {
+        ...state,
+        [`${payload?.name}`]: payload?.data,
```

Comment:
`[payload!.name]: payload!.data`

Reply:
Thanks, I didn't know about this option :)

Reply:
eslint marks as warning - Forbidden non-null assertion.eslint[@typescript-eslint/no-non-null-assertion]